<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QGroupBox" name="mSetGroupLang">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>270</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Язык программы</string>
   </property>
   <widget class="QRadioButton" name="mSetRus">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Русский</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="mSetEng">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Английский</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="mSetGroupHotkey">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>122</y>
     <width>270</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Горячие клавиши</string>
   </property>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>70</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>73</y>
      <width>70</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>98</y>
      <width>70</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Настройки:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>123</y>
      <width>70</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Справка:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>148</y>
      <width>70</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_open">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>25</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="p_open">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>22</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_save">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>73</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="p_save">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>70</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="p_settings">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>95</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_settings">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>98</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="p_help">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>120</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_help">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>123</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="p_exit">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>145</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_exit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>148</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="l_open_ro">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="p_open_ro">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>47</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>120</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть для чтения:</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="mSetSave">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>315</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mSetCancel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>315</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="mSetGroupTheme">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>65</y>
     <width>270</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Тема оформления</string>
   </property>
   <widget class="QRadioButton" name="mSetWhite">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Светлая</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="mSetGrey">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Темная</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
